Kelsey/schoollayout

kelseylam opened the pull request:
Added getfilter and filter class to placepredictionadapter and moved getpredictions from schoolformabstractfragment to placepredictionadapter to get the autocomplete delay to work using 250 ms delay.

charlesx2013 committed:
Revert "User Role for UserFragment and UserList"

charlesx2013 committed:
Revert "User Role for UserFragment and UserList"

kelseylam committed:
created new drawable white_circle_user_type to distinguish from white circles for announcements

kelseylam committed:
fixing school address autocomplete

kelseylam committed:
added getfilter and filter class to placepredictionadapter and moved getpredictions from schoolformabstractfragment to placepredictionadapter to get the autocomplete delay to work

charlesx2013 commented:
pls sentence better

charlesx2013 commented on sage-android/Sage/app/src/main/java/blueprint/com/sage/admin/browse/fragments/SchoolFormAbstractFragment.java:
```diff
@@ -165,20 +161,20 @@ public void initializeViews(Bundle savedInstanceState) {
-            public void afterTextChanged(Editable editable) {
-            }
-        });
+//        mSchoolAddress.addTextChangedListener(new TextWatcher() {
```
remove this!

charlesx2013 commented on sage-android/Sage/app/src/main/java/blueprint/com/sage/network/UrlConstants.java:
```diff
@@ -0,0 +1,11 @@
+package blueprint.com.sage.network;
+
+/**
+ * Created by kelseylam on 2/10/2016.
```
please remove this :(
this should be gitignored

charlesx2013 committed:
Revert "Revert "User Role for UserFragment and UserList""

This reverts commit 7e94c8afdc435f9ee50c3b6897f3e00b8b816022.

charlesx2013 committed:
Revert "Revert "User Role for UserFragment and UserList""

This reverts commit 6ac80c39d3c2b3982f1593b6da4f4949d220dbcb.

kelseylam committed:
took out commented code and rm --cached urlconstants

charlesx2013 committed:
Summary:
Fixing a bug with empty recylcer view

Test Plan:

Reviewers:

Subscribers: samz, kelseylam, baerica, amillman

Projects: #Sage-Mobile

charlesx2013 committed:
Revert "Revert "User Role for UserFragment and UserList""

This reverts commit 4c371c80e79a650663dd0e51d408749bd808defe.

charlesx2013 committed:
Summary:
Remove conflicts
Test Plan:

Reviewers:

Subscribers: samz, kelseylam, baerica, amillman

Projects: #Sage-Mobile
